hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/StorageVolume.java

```diff
@@ -548,6 +571,11 @@ public ConfigurationSource getConf() {
 
   public void failVolume() {
     setState(VolumeState.FAILED);
+    // Ensure it is set only once,
+    // which is the time when the failure was first detected.
+    if (failureDate == 0L) {
+      setFailureDate(Time.now());
```

Comment:
Let's use `Instant.now()` per HDDS-7911.

Reply:
@errose28 Can you help review this PR again? Thank you very much!